libraries/apollo-compiler/api/apollo-compiler.api

```diff
@@ -39,8 +39,31 @@ public final class com/apollographql/apollo3/compiler/CodegenMetadata$Companion
 
 public final class com/apollographql/apollo3/compiler/CodegenMetadataKt {
 	public static final fun schemaTypes (Lcom/apollographql/apollo3/compiler/CodegenMetadata;)Ljava/util/Set;
-	public static final fun toCodegenMetadata (Ljava/io/File;)Lcom/apollographql/apollo3/compiler/CodegenMetadata;
-	public static final fun writeTo (Lcom/apollographql/apollo3/compiler/CodegenMetadata;Ljava/io/File;)V
```

Comment:
Breaking changes everywhere in this file. I declare them OK cause the apollo-compiler API was not really meant to be accessed outside the Gradle Plugin.